lib/puppet/provider/rabbitmq_user_permissions/rabbitmqctl.rb

```diff
@@ -18,7 +18,7 @@ def self.users(name, vhost)
       run_with_retries do
         rabbitmqctl('-q', 'list_user_permissions', name)
       end.split(%r{\n}).each do |line|
-        line = .strip_backslashes(line)
+        line = strip_backslashes(line)
```

Comment:
@alexjfisher I think this change (and this commit) should resolve the issues mentioned in your review.